disco_aws_automation/disco_rds.py

```diff
@@ -133,9 +133,10 @@ def get_rds_security_group_id(self):
         """
         security_groups = self.disco_vpc_sg_rules.get_all_security_groups_for_vpc()
         for security_group in security_groups:
-            tags = tag2dict(security_group['Tags'])
-            if tags.get("meta_network") == "intranet":
-                return security_group['GroupId']
+            if security_group.get('Tags'):
+                tags = tag2dict(security_group['Tags'])
+                if tags.get("meta_network") == "intranet":
+                    return security_group['GroupId']
```

Comment:
I believe this should in disco_vpc, when does a security group doesn't have a tag?